.travis.yml

```diff
@@ -8,7 +8,8 @@ services:
 addons:
   apt:
     packages:
-      - postgresql-9.1-postgis-2.1
+      - postgresql-9.2
```

Comment:
It should be 

```
addons:
    postgresql: "9.2"
```

Reply:
https://docs.travis-ci.com/user/database-setup/#Using-a-different-PostgreSQL-Version

Reply:
Not sure if that has a different effect than installing the 9.2 version. According the to logs it used `9.2`. But let's try that! After all it is in the docs :smile: